verification-server-api/src/main/java/es/gob/radarcovid/verification/util/CheckSumUtil.java

```diff
@@ -27,10 +27,8 @@ public static int checkSum(String input) {
     }
 
     public static boolean validateChecksum(String validationCode) {
-        // Sample code - returns if code is correct
-        int last = Character.getNumericValue(validationCode.charAt(validationCode.length() - 1));
-        int checksum = checkSum(validationCode.substring(0, validationCode.length() - 1));
-        return last == checksum;
+        String dummy = validationCode.substring(0, validationCode.length() - 1)
```

Comment:
Is the assignment required if it will never be used?

Reply:
It was just for keep the side effects of the possible exceptions the same, but I also got bitten by substring method that makes the first index inclusive and the last exclusive (only noticed when reading the tests as I now rarely touch java

Reply:
Thanks. We prefer to maintain the use of checkSum method as long as if we change how it is calculated, we don't need to change other methods.